streams/id.m3u

```diff
@@ -136,6 +136,8 @@ http://vod.linknetott.swiftcontent.com/Content/HLS/Live/Channel(ch375)/index.m3u
 https://v3.siar.us/humabetangtv/live/playlist.m3u8
 #EXTINF:-1 tvg-id="IAmChannel.id",I AM CHANNEL (576p) [Not 24/7]
 http://iamchannel.org:1935/tes/1/playlist.m3u8
+#EXTINF:-1 tvg-id="BeritaSatuWorld.id",IDTV (720p) [Not 24/7]
+https://b1world.beritasatumedia.com/Beritasatu/B1World_manifest.m3u8
```

Comment:
It's already on the playlist: https://github.com/iptv-org/iptv/pull/14543/files#diff-78063f104290c770d5435f9f19044a8112a91bd59d6ebf6f3b05643f18d591e4R45

Reply:
Confirmed : https://github.com/iptv-org/iptv/blob/234cba4be90a7999423530a5bd399b705f531a13/streams/id.m3u#L44-L45